### PR TITLE
Lesson 5

### PR DESCRIPTION
**Pull request by Irina-pi**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -45,10 +45,76 @@ for (let i = 0; i < arr9.length; i++) {
...
+const arr10 = [2, 5, 8, 15, 0, 6, 20, 3];
 
 for (let i = 0; i < arr10.length; i++) {
     if (arr10[i] % 2 === 0) {
         console.log(arr10[i]);
     }
-}
\ No newline at end of file
+}
+
```
Comment (5tronciy) [inline main.js]: What is it for in lesson-5?

File: main.js [hunk 2]
```diff
@@ -45,10 +45,76 @@ for (let i = 0; i < arr9.length; i++) {
...
+];
+
+function seachByWorld(world = '') {
+    for (let i = 0; i < data.length; i++) {
+        result = data[i].country + ', ' + data[i].city + ', ' + data[i].hotel;
+        if (data[i].city === world || data[i].country === world || data[i].hotel === world) {
+            return result;
+        }
+    }
+}
```
Comment (5tronciy) [inline main.js]: It doesn't return the list of strings as required for the task.
Also, make sure the function doesn't return undefined.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Check your package.json. It has duplicated keys.
Remove node_modules.

File: main.js (hunk 1 above)
Comment (Irina-pi) [reply main.js]: I deleted extra code.

File: main.js (hunk 2 above)
Comment (Irina-pi) [reply main.js]: Yes, I made this function by task!

File: main.js [hunk 3]
```diff
@@ -1,54 +1,75 @@
...
+  const lengthFilterData = filterData.length;
+  let i = 0;
+  while (i < lengthFilterData) {
+    console.log((`${filterData[i].country}, ${filterData[i].city}, ${filterData[i].hotel}`));
+    i++;
+  }
+  if (lengthFilterData === 0) {
+    console.log('Search didn`t got results');
+  }
 }
```
Comment (5tronciy) [inline main.js:71]: The function returns `undefined` if we miss the return statement in it.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Ok, searchByWorld() always returns `undefined`. And excess file study-script.js was added.

File: main.js (hunk 3 above)
Comment (Irina-pi) [reply main.js:71]: I made it!

**Review by 5tronciy — APPROVED** [review]
It's the best attention.
Approve it, and I hope you will use the prettier next time.

**Pull request merged**